src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -35,20 +37,40 @@ export const AdvancedSearchController = ({
     resetAdvancedSearchFields, onSubmit, searchValue, setSearchValue, onMobileClose,
 }) => {
 
+    // TODO: see if it is better to change the URL only when we are submitting the form
+    // TODO: make it so the URL can be copied to the clipboard
+    const {
+        queryParams,
+        setJobType: actualSetJobType,
+        setJobDuration: actualSetJobDuration,
+        setFields: actualSetFields,
+        setShowJobDurationSlider: actualSetShowJobDurationSlider,
+        setTechs: actualSetTechs,
+        resetAdvancedSearchFields: actualResetAdvancedSearchFields,
+        setSearchValue: setUrlSearchValue,
```

Comment:
Why doesn't the simple names on the left work? I honestly didn't think too much, it's faster to ask

Reply:
The names on the left are the same names as the props passed in to the controller, which relate to the "normal" functions. It was just a way to differentiate the names of the "normal" and "URL-changing" functions.